readme.md

```diff
@@ -14,7 +14,7 @@ yarn add svelte parcel-bundler parcel-plugin-svelte @babel/polyfill -D
 
 ### Update the package.json
 
-Update your `package.json` to contain these scripts (for more information on these commands, see: https://parceljs.org/cli.html):
+Update your `package.json` to contain these scripts (for more information on these commands, see: https://parceljs.org/features/cli/):
```

Comment:
This plugin is for v1, so should probably be this:

```suggestion
Update your `package.json` to contain these scripts (for more information on these commands, see: https://v1.parceljs.org/cli.html):
```

Reply:
Oh, I wasn't aware of that. 
I think it's wroth to add an indication of that in the readme. Been battling with errors thrown by using v2 for the 30 mins :)